commit f55bfe72ecdb31942f734fa2582f3129d9cd1b79
Author: Sergei Trofimovich <siarheit@google.com>
Date:   Thu Apr 7 21:15:19 2016 +0100

    Language/PureScript/Ide/Types.hs: fix haddock markup
    
    Failed as:
    
        src/Language/PureScript/Ide/Types.hs:63:7: parse error on input ‘(’
    
    Signed-off-by: Sergei Trofimovich <siarheit@google.com>

diff --git a/src/Language/PureScript/Ide/Types.hs b/src/Language/PureScript/Ide/Types.hs
index d1349e1..54e2fc5 100644
--- a/src/Language/PureScript/Ide/Types.hs
+++ b/src/Language/PureScript/Ide/Types.hs
@@ -59,9 +59,9 @@ data ExternDecl
         [DeclIdent] -- The exported identifiers
     -- | A data/newtype declaration
     | DataConstructor
-      DeclIdent -- ^ The type name
+      DeclIdent -- The type name
       (P.ProperName 'P.TypeName)
-      P.Type      -- ^ The "type"
+      P.Type      -- The "type"
     -- | An exported module
     | TypeClassDeclaration (P.ProperName 'P.ClassName)
     | Export ModuleIdent -- The exported Modules name
